### DISCUSSION TITLE
Added a drawsvg example to my Markdown Exec gallery

### DISCUSSION THREAD

**Discussion by pawamoy** [cduck/drawSvg#131] [Show and tell]
Markdown-Exec is an extension for Python-Markdown that allows to execute code blocks in Markdown pages :slightly_smiling_face: 

I added an example that shows how to generate an SVG drawing with drawsvg to my gallery of examples: https://pawamoy.github.io/markdown-exec/gallery/#with-drawsvg-2.

That's it! Happy to get feedback, or more drawsvg examples :smile:

**Comment by cduck** [comment]
Thanks for sharing!
